node/src/Commands.ts

```diff
@@ -3812,6 +3843,11 @@ export interface BaseScanOptions {
      * represent the results as compact single-allocation packed encoding.
      */
     readonly count?: number;
+    /**
+     * The type of the object to scan.
+     * Types are the data types of Valkey.
+     */
+    type?: ObjectType;
```

Comment:
This will add `type` for `ZSCAN` options, and if user will add it - it will fail the command.
Create a new type/interface to avoid that.